2023-Oct-18/Dave/main.py

```diff
@@ -0,0 +1,94 @@
+import random
+from word_grid import WordGrid
+from point import Point
+from direction import Direction
+from placement import Placement
+
+
+def main():
+    word_list = ["MEOW", "CAT", "WOOF", "DOG"]
+
+    word_search = generate_word_search(
+        rows=4, cols=4, word_list=word_list)
```

Comment:
> Could you explain the 'enforce it with , *,' part?

```Python
def func(*, keywordEnforcedArg:int):
    ...

func(1)
``` Throws (and I get a red squiggle)

Reply:
Useful, but I don't think it's what I want here. The keyword operators are just for clarity as I'm passing literals rather than named variables and it helps me to see that rows and columns aren't getting mixed up.

But I take your point from yesterday about `word_list=word_list` being too verbose, so I think that's fine to get rid of.

Reply:
Damn, just realised that this isn't straight-forward. Positional arguments can't come after keyword arguments. Means either rewriting the function signature or just accepting a little repetition. For now, I think I prefer repetition. Seems too minor to be caught up on.